lib/std/unicode.zig

```diff
@@ -23,11 +23,12 @@ pub fn utf8CodepointSequenceLength(c: u21) !u3 {
 /// returns a number 1-4 indicating the total length of the codepoint in bytes.
 /// If this byte does not match the form of a UTF-8 start byte, returns Utf8InvalidStartByte.
 pub fn utf8ByteSequenceLength(first_byte: u8) !u3 {
-    return switch (@clz(u8, ~first_byte)) {
```

Comment:
I recall in previous benchmarks `@clz` was fastest. Could you maybe share results with different optimization levels; and perhaps different cpus and architectures?

Reply:
I've updated the 1st table.

Reply:
> pure ASCII strings 	6729 MiB/s [80] 	8990 MiB/s [80] 	8501 MiB/s [80] 	7485 MiB/s [80]

curious.

Reply:
> I recall in previous benchmarks `@clz` was fastest.

How did you benchmark it? What kind of data?

Reply:
In #3987 and by using `lib/std/unicode/throughput_test.zig`